### DISCUSSION TITLE
Uploading Word file in Turninit using SEB

### DISCUSSION THREAD

**Discussion by vanie76** [SafeExamBrowser/seb-win-refactoring#206] [Q&A]
Trying to upload word file in Turnitin (in Moodle) using SEB. 
![image](https://user-images.githubusercontent.com/87995554/134304436-e4fb4658-1991-4a41-972f-b463473279d8.png)

Manage to configure the word application in SEB. Manage to go until to choose file option but unableto click the button to upload the file
![image](https://user-images.githubusercontent.com/87995554/134304555-b6c5fbe0-254d-4861-b4ca-7afd5b656516.png).
Anyone have try this before. Can you please share the configuration file.
Please advice if my setting is incorrect

Assessment task
Student have to open the Word application. Type their answers , save it and upload in Turnitin..

Tried enabling the option below but failed
![image](https://user-images.githubusercontent.com/87995554/134330668-8c8210ff-8c45-4d46-a3b3-8ba9c5512b89.png)

**Comment by dbuechel** [comment]
If you'd like to choose the default download directory for Windows, you'd need to enter e.g. `%UserProfile%\Downloads` in the text field.

**Comment by gammagamer** [comment]
Is Word secure? Did you manage somehow to block Word from accessing other files, other than what's in the download map?

**Reply by dbuechel** [reply]
No, we can't do that. Once you allow Word as third-party application, it's full feature set will be available to the users (unless you use managed computers where you have control over the Word installation).
